frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx

```diff
@@ -349,7 +349,10 @@ export default class LineAreaBarChart extends Component {
       settings,
     } = this.props;
 
-    const orderedSeries = getOrderedSeries(series, settings);
+    const isReversed =
+      settings["stackable.stack_type"] !== null &&
+      settings["stackable.stack_type"] !== undefined;
```

Comment:
I'm not usually a big proponent of comments, but I think one could be helpful here to explain why we would want to be reversing stacked charts.

Reply:
Definitely agree. I usually prefer when code is self-explanatory, but in this case I think explaining the business/design logic would be helpful.